ush/load_fv3gfs_modules.sh

```diff
@@ -40,6 +40,8 @@ else
   echo WARNING: UNKNOWN PLATFORM
 fi
 
+module list
```

Comment:
Is this a debug print you want to remove, or did you want it to be permanent?

Reply:
no.  I would like to see what the loaded modules are in a job upfront.